xamarin/Xamarin.Forms
# Added memory leak gallery

### Description of Change ###

Checking the correct disposing Navigation page and all content inside.

### Issues Resolved ### 


### API Changes ###

 
 None

### Platforms Affected ### 


- Core (all platforms)

### Behavioral/Visual Changes ###


None

### Before/After Screenshots ### 


Not applicable

### Testing Procedure ###

- run Memory leak Gallery
- click on any button twice
- click on `Check Result` button
- check result on top label of menu 

### PR Checklist ###
